src/stylesheet-registry.js

```diff
@@ -210,6 +211,33 @@ export default class StyleSheetRegistry {
       return acc
     }, {})
   }
+
+  toReact(options = {}) {
+    return this.cssRules().map(args => {
+      const id = args[0]
+      const css = args[1]
+      return React.createElement('style', {
+        id: `__${id}`,
+        // Avoid warnings upon render with a key
+        key: `__${id}`,
+        nonce: options.nonce ? options.nonce : undefined,
+        dangerouslySetInnerHTML: {
+          __html: css
+        }
+      })
+    })
+  }
+
+  toHTML(options = {}) {
```

Comment:
I don't think that it these two method should be in the StylesheetRegistry as they are mostly for SSR - I'd revert the changes to `src/server.js`. We can save a tiny bit in client size bundle size. 

`src/style.js` could export the StylesheetRegistry singleton to flush the default when needed and these methods should accept an instance of StylesheetRegistry to flush.

By the way I think we need to `flush` instead of just callin `cssRules()`

Reply:
Why would you need to flush? Just discard the registry instance and let garbage collection do its thing.

Reply:
As for moving those functions: https://martinfowler.com/bliki/TellDontAsk.html

Reply:
If we keep the singleton we'd need to flush. 

I agree with you that the singleton is not cool but honestly we should aim to keep the DX good and just enable advanced stuff for the ones who are more experienced.

I'll let @timneutkens decide on this anyway